src/plugins/credential_management/common/index.ts

```diff
@@ -4,4 +4,4 @@
  */
 
 export const PLUGIN_ID = 'credentialManagement';
-export const PLUGIN_NAME = 'credentialManagement';
+export const PLUGIN_NAME = 'Credential Management';
```

Comment:
Np: If I remembered correctly, the plugin names are following "snake_case"? Tho this won't be an issue for internal plugins, but for external plugins, it will affect the artifact naming. The release infra actually depends on the naming for release stand-alone plugin artifacts.

Reply:
> Np: If I remembered correctly, the plugin names are following "snake_case"? Tho this won't be an issue for internal plugins, but for external plugins, it will affect the artifact naming. The release infra actually depends on the naming for release stand-alone plugin artifacts.

Updated